src/components/map/layers/NodeLayer.tsx

```diff
@@ -43,31 +49,40 @@ export default class NodeLayer extends Component<MarkerLayerProps> {
             type = node.type;
         }
 
+        if (node.stop && node.stop.hastusId && this.props.mapStore!.zoom >= HASTUS_MIN_ZOOM) {
+            children.push(
+                `<div class=${s.hastusId}>${node.stop.hastusId}</div>`,
+            );
+        }
```

Comment:
I would take this out from getIcon() method and put it into a getLabel() method instead.

Reply:
That sounds like a good idea

Reply:
fixed